appcues/src/main/java/com/appcues/AppcuesViews.kt

```diff
@@ -0,0 +1,100 @@
+package com.appcues
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.FrameLayout
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+
+/**
+ * Base class for Appcues Compositions, provides a similar shell as native ComposeView but it adds internal
+ * flexibility since ComposeView is final.
+ */
+public abstract class AppcuesComposeView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : FrameLayout(context, attrs, defStyleAttr) {
```

Comment:
using FrameLayout now instead of AbstractComposeView, but that entails adding some extra restraints on addView(***) methods, same thing that AbstractComposeView does to ensure no other view is added here.

I'm also adding and removing everytime we set/clear content to make sure that customers wont removeAllViews or removewView(index = 0) and break our Custom Layout